src/editors/containers/ProblemEditor/index.test.jsx

```diff
@@ -62,17 +63,27 @@ describe('ProblemEditor', () => {
     test('assets loaded, block and studio view not yet loaded, Spinner appears', () => {
       expect(shallow(<ProblemEditor {...props} assetsFinished />)).toMatchSnapshot();
     });
+    test('advanceSettings loaded, block and studio view not yet loaded, Spinner appears', () => {
+      expect(shallow(<ProblemEditor {...props} advanceSettingsFinished />)).toMatchSnapshot();
```

Comment:
Same as the other comment - I would feel a bit more comfortable if we at least tested here that an actual spinner appears (find in shallow output) rather than adding a snapshot.
While we're at it... same problem for the tests above. Since they are basically exactly the same test logic, maybe if you test the behavior in this test, can you just copy it also to the tests above?
By the way I don't have a problem with leaving the snapshots, in principle. My suggestion is just a little test case to add to this.
`const wrapper = shallow(...)`
`expect(wrapper.find(Spinner)).toBe(true)` (I don't know if it returns true or what the return value is, but something like that.)